projects/plugins/jetpack/changelog/rnmobile-videopress-block-add-playback-settings

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: other
+Comment: Updated composer.lock.
```

Comment:
I'm not the ideal person to say this, but still, the comment should reflect the changes introduced here :-)

Reply:
@retrofox, I didn't add this, it was automatically generated. :/

Reply:
@SiobhyB Not sure if the last commits were needed after merging `trunk` into this branch, I'm wondering why we need to upgrade the VideoPress package version and upgrade the version in the Forms package 🤔.

Reply:
@fluiddot, I asked about this more at p1679412895593819-slack-C03TA48NSUX, which might be a good place to ask any extra questions from the Jetpack team.

Reply:
I'm not sure why it is necessary to update the versions here, but sometimes when opening a PR right after some project was updated we need to update its version for the next alpha.
There was a `minor` VideoPress package update very recently, so maybe that is the reason? :man_shrugging: 

In any case, the `Linting / Changelogger validity (pull_request)` test fails whenever it detects that some version change is needed for any reason.